crates/cli-support/src/js/rust2js.rs

```diff
@@ -528,10 +543,24 @@ impl<'a, 'b> Rust2Js<'a, 'b> {
         ret.push_str(") {\n");
         ret.push_str(&self.prelude);
 
-        let mut invoc = self.ret_expr.replace(
-            "JS",
-            &format!("{}({})", invoc, self.js_arguments.join(", ")),
-        );
+        let mut invoc = if self.variadic {
+            if self.js_arguments.is_empty() {
+                unreachable!("the last argument of a variadic must be a slice");
```

Comment:
Should not it be `bail_span!` instead of `unreachable!`? And the text is a bit outdated, probably should say something like "variadic functions should have at least 1 argument".

Reply:
I think that before this was unreachable, but since I removed the checking code it now might not be. I'll check.